ReactiveCocoa/Swift/UnidirectionalBinding.swift

```diff
@@ -0,0 +1,313 @@
+	@discardableResult
+	public static func <~ <Source: SignalProtocol>(target: Self, signal: Source) -> Disposable? where Source.Value == ValueType.Wrapped, Source.Error == NoError {
+		return target <~ signal.map(ValueType.init(reconstructing:))
+	}
+
+	/// Binds a producer to a target, updating the target's value to the latest
+	/// value sent by the producer.
+	///
+	/// - note: The binding will automatically terminate when the target is
+	///         deinitialized, or when the producer sends a `completed` event.
+	///
+	/// ````
+	/// let property = MutableProperty(0)
+	/// let producer = SignalProducer<Int, NoError>(value: 1)
+	/// property <~ producer
+	/// print(property.value) // prints `1`
+	/// ````
+	///
+	/// ````
+	/// let property = MutableProperty(0)
+	/// let producer = SignalProducer({ /* do some work after some time */ })
+	/// let disposable = (property <~ producer)
+	/// ...
+	/// // Terminates binding before property dealloc or
+	/// // signal's `completed` event.
+	/// disposable.dispose()
+	/// ````
+	///
+	/// - parameters:
+	///   - target: A target to be bond to.
+	///   - producer: A producer to bind.
+	///
+	/// - returns: A disposable that can be used to terminate binding before the
+	///            deinitialization of the target or the producer's `completed`
+	///            event.
+	@discardableResult
+	public static func <~ <Source: SignalProducerProtocol>(target: Self, producer: Source) -> Disposable where Source.Value == ValueType.Wrapped, Source.Error == NoError {
+		return target <~ producer.map(ValueType.init(reconstructing:))
+	}
+
+	/// Binds a property to a target, updating the target's value to the latest
+	/// value sent by the property.
+	///
+	/// - note: The binding will automatically terminate when either the target or
+	///         the property deinitializes.
+	///
+	/// ````
+	/// let dstProperty = MutableProperty(0)
+	/// let srcProperty = ConstantProperty(10)
+	/// dstProperty <~ srcProperty
+	/// print(dstProperty.value) // prints 10
+	/// ````
+	///
+	/// ````
+	/// let dstProperty = MutableProperty(0)
+	/// let srcProperty = ConstantProperty(10)
+	/// let disposable = (dstProperty <~ srcProperty)
+	/// ...
+	/// disposable.dispose() // terminate the binding earlier if
+	///                      // needed
+	/// ````
+	///
+	/// - note: The binding will automatically terminate when either property is
+	///         deinitialized.
+	///
+	/// - parameters:
+	///   - target: A target to be bond to.
+	///   - property: A property to bind.
+	///
+	/// - returns: A disposable that can be used to terminate binding before the
+	///            deinitialization of the target or the source property.
+	@discardableResult
+	public static func <~ <Source: PropertyProtocol>(target: Self, property: Source) -> Disposable where Source.Value == ValueType.Wrapped {
+		return target <~ property.producer
+	}
+}
+
+/// A type-erased view of a binding consumer.
+public final class AnyBindingTarget<Value>: BindingTarget {
```

Comment:
You mentioned that this was destined to work with the `Rex` bindings. I think we should leave this out until Rex is integrated with RAC. I dislike having untested code and it's not really essential to this PR.